### PR TITLE
Adding Contact Section

### PR DESCRIPTION
**Pull request by okemboi**
What does this PR do?
1. Creating Contact section

## How can this changes be manually tested
1. Clone this project
3. Navigate into cloned Project
4. run `npm install`
5. open index.html in browser

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -241,7 +241,7 @@ <h3 class="frameworks">Skills</h3>
         </div>
     </section>
 
-    <section class="contact-form">
+    <section class="contact-form" id="Contact">
         <p class="footer-text">
             I'm always interested in hearing about new projects, so if you'd like to
             chat please get in touch.
```
Comment (brainconnect93) [inline index.html:247]: - [ ] Great effort working on your contact form project, but currently I could not submit some data to the form, Kindly check the name attribute you have given the input form and be sure they are correct, Also the styling still requires some attention in order for it to look like what we have in the original design. Kindly work on it and be sure to have a perfect project before resubmitting it for review.

 Below is a video explanation 👇 

[screen-capture (1).webm](https://github.com/okemboi/portfolio/assets/78634841/5530f9b4-8afc-4ac2-b5e9-434708010c6b)

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @okemboi, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight 🥇

- Descriptive Pull Request title and summary. ✔️
- Form uses the action and method attribute to integrate with [formspree service](https://formspree.io/html). ✔️
- Input fields have the required attribute to activate the validation ✔️
- Max-length attribute Used on the input ✔️
- Pair-programming collaborative activity ✔️
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -241,27 +241,28 @@ <h3 class="frameworks">Skills</h3>
...
         </p>
         <form class="form-content" action="https://formspree.io/f/mvonwygw" method="POST">
-            <input type="text" name="full_name" maxlength="30" placeholder="Full name" required />
 
             <div class="first-last">
```
Comment (Alusp) [inline index.html:251]: - [ ] Great job in implementing the changes requested by the previous reviewer, everything is working as it should, however, on the mobile version, the `full-name` section is not focusable which means its hidden on the mobile view
- [ ] Possible solution would be to remove the display properties in `.first-last` class on mobile view and adjust accordingly. :slightly_smiling_face: 
- [ ] As mentioned by the previous reviewer, Please ensure that your design matches the given design, matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 

The example below shows the difference between yours and the accepted one: :arrow_down: 

 <table>
    <tr>
      <td>Yours </td>
      <td>Figma</td>
    </tr>
    <tr>
      <td>

 
Student Form
 
</td>
      <td>

Figma Form

</td>
    </tr>
  </table>

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @okemboi, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 close

 ## Highlights

- [ ] Linters set-up ✔️
- [ ] Commit messages descriptive ✔️
- [ ] Professional GitHub Flow used ✔️
- [ ] Profesional PR Summary ✔️
- [ ] Use Form-spree :heavy_check_mark: 
- [ ] Use HTML form Validation :heavy_check_mark: 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alusp in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Johnadibe) [reply index.html:251]: > - [ ] I observed that the previous request made by the code reviewer has not been implemented yet. Specifically, on the mobile version, the full-name section remains non-focusable, indicating that it is still hidden in the mobile view. To address this, the suggested solution is to remove the display properties in the .first-last class for the mobile view and make the necessary adjustments.😊
> 
> - [ ] Additionally, I noticed that your current design still does not align with the given design requirements. Aligning your design with the provided design specifications is crucial as it enhances the likelihood of creating a successful and comprehensive design solution that fulfills user needs, adheres to regulations, and operates efficiently. I kindly urge you to take these recommendations into consideration and make the necessary updates accordingly.👍
> 
> Figma design:
> 
> at
> 
> Your design:
> 
> at
>

**Review by Johnadibe — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️
### Hi @okemboi and @Reconfort🙋‍♂️, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
- No linter error ✔️
- Form uses form spree service✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tugsem — APPROVED** [review]
## STATUS: APPROVED 👍🎯  
Hello @okemboi 👋 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⭐ 
✔️ text aligned to the center and input for full name is visible now
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**